org.elysium.parent/org.elysium.ui/src/org/elysium/ui/Activator.java

```diff
@@ -20,9 +17,6 @@
 	public void start(BundleContext context) throws Exception {
 		super.start(context);
 		instance = this;
-		OutdatedMarkerAdder outdatedMarkerAdder = getInjector(ORG_ELYSIUM_LILYPOND).getInstance(OutdatedMarkerAdder.class);
```

Comment:
I think the outdated marker should not depend on whether the LilyPond compilation is active during build. After this change, if you disable LilyPond compilation, the outdated marker won't remind you that you should recompile the changed files.

Reply:
No, because with this flag you only deactivate the Compiler. The LilyPondBuilder is still active (as long as the XtextNature is active). The LilyPondBuilder will still check the delta, mark files whose index entry has changed (and only those) as dirty and call the compiler - which then does nothing.

The "file system" delta may fire a file change event even if the content really has not changed at all (e.g. syntax update which did not modify the file).